aspnetcore/mvc/views/tag-helpers/intro.md

```diff
@@ -151,6 +151,26 @@ The preceding code can be written as:
        disabled="@(Model?.LicenseId == null)" />
 ```
 
+## Tag helper initializers
+
+While attributes can be used to configure individual instances of tag helpers, `<xref:Microsoft.AspNetCore.Mvc.Razor.ITagHelperInitializer />` can be used to configure all tag helper instances of a specific kind. Consider the following example of a tag helper initializer that configures the `asp-append-version` attribute / `AppendVersion` property for all `ScriptTagHelpers` in the app:
```

Comment:
@Rick-Anderson I missed pointing this out earlier, but how does one link to a generic type? Here's the type in question: https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.mvc.razor.itaghelperinitializer-1?view=aspnetcore-6.0

Reply:
```suggestion
While attributes can be used to configure individual instances of tag helpers, `[`ITagHelperInitializer`](/dotnet/api/microsoft.aspnetcore.mvc.razor.itaghelperinitializer-1) can be used to configure all tag helper instances of a specific kind. Consider the following example of a tag helper initializer that configures the `asp-append-version` attribute / `AppendVersion` property for all `ScriptTagHelpers` in the app:
```

Reply:
That's the easy way. You can also do it from VS Code with the [Docs Authoring Pack](https://docs.microsoft.com/en-us/contribute/how-to-write-docs-auth-pack)